### PR TITLE
Math-magicians website using React test library

### PR DESCRIPTION
**Pull request by nganifaith**
-- Add test for Home page, Calculator page, Quote page by using jest snapshots. 

-- Add unit test for calculate.js file by testing all maths functions that is 
- Addition
- Multiplication
- Division
- Subtraction
- Percentage
- and +/-

### CODE REVIEW

File: src/__tests__/Calculate.test.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  const result = Calculate(data, buttonName);
+  expect(result.total).toBe('25');
+});
+
+test('should calculate the division', () => {
+  const buttonName = '=';
+  const data = { total: '5', operation: '/', next: '5' };
+  const result = Calculate(data, buttonName);
+  expect(result.total).toBe('1');
+});
```
Comment (rindrajosia) [inline src/__tests__/Calculate.test.js:36]: - [x] [REQUIRED] Please add test for division by 0

File: src/Logic/operate.js [hunk 2]
```diff
@@ -30,7 +30,7 @@ const Operate = (number1, number2, operation) => {
       }
       break;
     case '%':
-      results = numberTwo.div(100);
+      results = numberOne.div(100);
```
Comment (rindrajosia) [inline src/Logic/operate.js:33]: - [x] [REQUIRED] I got an error related to the prop-types on my console when I perform this operation (-/+). please fix it

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Hi, 

Good job so far!
There are some issues to revise before continuing to the next project but hang in there and read the comments below with care. You can do it! :muscle:


## Required Changes ♻️

- [ ] Please update your live demo link and make sure that all links in the readme redirect to the correct page like the issue link. Keep in mind that your readme is the first thing users check.
- [ ] all functionalities should be tested by using [React Testing Library](https://testing-library.com/docs/react-testing-library/intro/), @testing-library/react, see the [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/testing/useReactTestingLibrary.png) from [Pathwright](https://microverse.pathwright.com/library/react-redux-v11/184438/path/step/106580952/).

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by afanimartin — APPROVED** [review]
## Approved 👍🏿 👍🏿 👍🏿 
Nice work making the requested changes. Merge and proceed to the next project.

### Martin Afani

**Pull request merged**